src/main/java/kai/commands/CreateDeadlineCommand.java

```diff
@@ -0,0 +1,46 @@
+package kai.commands;
+
+import java.time.LocalDate;
+
+import kai.TaskList;
+import kai.tasks.Deadline;
+import kai.ui.Ui;
+
+
+/**
+ * This Command creates a Deadline and adds it to the TaskList
+ * It is immediately implemented as one-use
+ * to avoid potential re-use issues down the line
+ */
+public class CreateDeadlineCommand extends Command {
```

Comment:
Should class have noun names rather than verbs?

Reply:
~Pretend~ I failed my English Classes: How would an example look like? Thanks!